src/matchers/toReject/predicate.js

```diff
@@ -0,0 +1,10 @@
+export default async promise => {
```

Comment:
I don't think the `async` is needed here as we aren't awaiting anything. Also the `resolved`/`rejected` values aren't being used, how about something like:

```js
export default promise => promise.then(() => false, () => true);
```